user/forgejo-runner/template.py

```diff
@@ -0,0 +1,51 @@
+pkgname = "forgejo-runner"
+pkgver = "3.5.1"
+pkgrel = 0
+build_style = "makefile"
+make_build_target = "all"
+make_check_target = "test"
+make_use_env = True
+hostmakedepends = ["go"]
+depends = ["podman"]
+pkgdesc = "Daemon that runs CI jobs for Forgejo"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "MIT"
+url = "https://code.forgejo.org/forgejo/runner"
+source = f"{url}/archive/v{pkgver}.tar.gz"
+sha256 = "0c53e5ee6ea57eadcc3ebbb4fbe203478500f78b058b13d8ccc3b63b775955c0"
+
+
+def prepare(self):
+    from cbuild.util import golang
+
+    golang = golang.Golang(self).mod_download()
+
+
+def init_build(self):
+    from cbuild.util import golang
+
+    self.make_env.update(golang.get_go_env(self))
+    self.make_env["EXTRA_GOFLAGS"] = f"{self.get_goflags(shell=True)} -trimpath"
+    # https://github.com/golang/go/issues/64875
+    self.make_env["CGO_ENABLED"] = "1"
+    self.make_env["RELEASE_VERSION"] = pkgver
+    self.make_env["LDFLAGS"] = (
+        f"-X gitea.com/gitea/act_runner/internal/pkg/ver.version={pkgver} -linkmode=external"
+    )
+
+
+def check(self):
+    from cbuild.util import golang
+
+    golang = golang.Golang(self).check()
+
+
+def install(self):
+    self.install_bin("forgejo-runner")
+    self.install_file(
+        self.files_path / "docker_host.env", "usr/share/forgejo-runner"
+    )
+    self.install_service(self.files_path / "forgejo-runner")
+    self.install_tmpfiles(self.files_path / "tmpfiles.conf")
+
+    self.install_license("LICENSE")
```

Comment:
```suggestion
pkgname = "forgejo-runner"
pkgver = "3.5.1"
pkgrel = 0
build_style = "go"
make_check_args = [
    "-skip",
    "Test_runCreateRunnerFile|Test_ping",
    "./...",
]
hostmakedepends = ["go"]
depends = ["podman"]
pkgdesc = "Daemon that runs CI jobs for Forgejo"
maintainer = "Gnarwhal <git.aspect893@passmail.net>"
license = "MIT"
url = "https://code.forgejo.org/forgejo/runner"
source = f"{url}/archive/v{pkgver}.tar.gz"
sha256 = "0c53e5ee6ea57eadcc3ebbb4fbe203478500f78b058b13d8ccc3b63b775955c0"


def install(self):
    self.install_bin(
        f"{self.cwd}/{self.make_dir}/act_runner", name="forgejo-runner"
    )
    self.install_file(
        self.files_path / "docker_host.env", "usr/share/forgejo-runner"
    )
    self.install_service(self.files_path / "forgejo-runner")
    self.install_tmpfiles(self.files_path / "tmpfiles.conf")
    self.install_license("LICENSE")
```

and delete both the patches

Reply:
feel free to add make_build_args for the `f"-ldflags=-X gitea.com/gitea/act_runner/internal/pkg/ver.version=v{pkgver}"` too

Reply:
Unreal...